guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<p>Change:</p><p>"<code>TimeSeriesModel</code> supports training model for both univariate as well as multivariate cases. "</p><p>to</p><p>"<code>TimeSeriesModel</code> has support for both univariate as well as multivariate time series. "</p><p> </p><p>In the next paragraph, replace "In multivariate cases,", with "In the case of multivariate time series," and "In univariate cases, " with "In the case of univariate time series, "</p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/953/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>